core/operation/src/main/java/uk/gov/gchq/gaffer/named/operation/AddNamedOperation.java

```diff
@@ -51,7 +52,7 @@
  * A {@code AddNamedOperation} is an {@link Operation} for creating a new {@link NamedOperation}
  * and adding it to a Gaffer graph.
  */
-@JsonPropertyOrder(value = {"class", "operationName", "description", "score", "operations"}, alphabetic = true)
+@JsonPropertyOrder(value = {"class", "operationName", "description", "score", "labels", "operationChain", "input", "overwriteFlag", "parameters", "readAccessRoles", "writeAccessRoles", "readAccessPredicate", "writeAccessPredicate"}, alphabetic = true)
```

Comment:
Was this ordering required anything, is it neater than remaining being alphabetical order?